docs/resources/aws_iam_access_keys.md

```diff
@@ -0,0 +1,164 @@
+---
+title: About the aws_iam_access_keys Resource
+---
+
+# aws_iam_access_keys
+
+Use the `aws_iam_access_keys` InSpec audit resource to test properties of all or multiple IAM Access Keys.
+
+To test properties of a Access Key, use the `aws_iam_access_key` resource.  You may also test properties of the access keys of a particular user using the `aws_iam_user` resource.
+
+Access Keys are tightly coupled to the AWS User resources.  This resource is best used when you need so perform audits of all keys, or keys specified by criteria that don't relate to any particular user.  
+
```

Comment:
Access Keys are related to the AWS User resources.  Use this resource *to* perform audits of all keys or of keys specified by criteria unrelated to any particular user. [consider an example?]